content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,120 @@
+
+First, navigate to the **New Project** tab in the Pulumi Cloud console. Choose your source template from Pulumi, then you can optionally choose your cloud and language to filter the results or search all templates.
+
+![Animation of how to create a new project in Pulumi Cloud](./pulumi-create-new-project.gif)
+
+{{% notes type="info" %}}
+If your Pulumi administrator has configured [custom templates](/docs/pulumi-cloud/developer-portals/templates) for your organization, you will be able to choose from your organization's custom templates from your own image source.
+If you select "Use a template" but your organization doesn't have custom templates, you can choose one of Pulumi's public templates to get started.
+{{% /notes %}}
+
+Type `aws-python` or `aws-typescript` and click **next**.
+
+### Project details
+
+Provide a name for the project, along with an optional description and the stack name which will be set in the resulting `Pulumi.yaml` file and the name of `Pulumi.<stack>.yaml`.
+
+### Configuration
+
+Provide any values required for the configuration declared in the [template](/docs/pulumi-cloud/developer-portals/templates), in this example, you can set the **aws:region** you would like to deploy into.
+
+These values populate the `config` section in the `Pulumi.<stack>.yaml` file, specifying the necessary settings for your project.
+
+### Environment
+
+If you've configured [environments](/docs/pulumi-cloud/esc) with your organization, you can specify one to use with the resulting stack. Environments enable teams to manage hierarchical collections of configuration and secrets without needing to re-specify all of them during project creation.
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end and can be set on each stack. However, by creating an environment with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+### Repository
+
+Configure the repository and optional subdirectory to use when committing your new project code.
+
+{{% notes "info" %}}
+If you granted the Pulumi GitHub app access to _all_ repositories, the New Project Wizard will allow users to create projects with Deployments enabled in new repositories.
+
+If the app only has access to _some_ repositories, users will only be able to create new projects with Deployments enabled in _existing_ repositories.
+{{% /notes %}}
+
+The GitHub owner is not configurable, as that must match the Pulumi GitHub app's owner in order to work with Deployments.
+
+### Deployment settings
+
+Configure the behavior of Deployments, including when to trigger new Deployments and any environment variables required for the Pulumi program to run.
+
+A full description of each setting is available [here](/docs/pulumi-cloud/deployments/reference/#deployment-settings).
```

Comment:
Technically fine, but it's better for both SEO and usability to link using relevant text  rather than "click [here](#)".

Reply:
thx